### DISCUSSION TITLE
Out now: spaCy v3.0

### DISCUSSION THREAD

**Discussion by ines** [explosion/spaCy#6877] [News & Announcements]
It's finally here 🎉 Give it a try and let us know what you think!

## 💫 spaCy v3.0 in a nutshell

The new version features all new **transformer-based pipelines** that bring spaCy's accuracy right up to the current **state-of-the-art**. You can use any pretrained transformer to train your own pipelines, and even share one transformer between multiple components with **multi-task learning**. Training is now fully configurable and extensible, and you can define your own custom models using **PyTorch**, **TensorFlow** and other frameworks. The new spaCy projects system lets you describe whole **end-to-end workflows** in a single file, giving you an easy path from prototype to production, and making it easy to clone and adapt best-practice projects for your own use cases.

## 📖 Read more

- [**v3.0.0 release notes**](https://github.com/explosion/spaCy/releases/tag/v3.0.0): Official GitHub release 
- [**New in v3.0**](https://spacy.io/usage/v3#migrating): New features, backwards incompatibilities and migration guide.
- [**Installation Quickstart**](https://spacy.io/usage): Install the new version, pipelines and add-ons for your specific setup.
- [**Training Quickstart**](https://spacy.io/usage/training#quickstart): Generate a training config for your specific use case.
- [**Benchmarks**](https://spacy.io/usage/facts-figures#benchmarks): Results and accuracy comparisons.
- [**Projects & Project Templates**](https://spacy.io/usage/projects): Get started by cloning a project template.
- [**Migration guide from v2.x to v3.x**](https://spacy.io/usage/v3#migrating): How to update your code
- [**Introducing spaCy v3.0**](https://explosion.ai/blog/spacy-v3): Blog post introducing the release
- [**Videos**](https://youtube.com/c/ExplosionAI): Introductions, walkthroughs and in-depth tutorials
